pkgs/by-name/cl/clash-nyanpasu/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, clash-verge
+, fetchurl
+}:
+clash-verge.overrideAttrs (old: rec {
+  pname = "clash-nyanpasu";
+  version = "1.4.5";
+
+  src = fetchurl {
+    url = "https://github.com/keiko233/clash-nyanpasu/releases/download/v${version}/clash-nyanpasu_${version}_amd64.deb";
+    hash = "sha256-cxaq7Rndf0ytEaqc7CGQix5SOAdsTOoTj1Jlhjr5wEA=";
+  };
+
+  # use builtin clash binaries, as clash-meta in nixpkgs is outdated
```

Comment:
Does the comment still apply on master?

If so, please move that comment above `postFixup`.